example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
```

Comment:
He seems to have nothing to do with the function modification, should you delete it